app/assets/javascripts/application.js

```diff
@@ -1 +1,21 @@
-//= require header-footer-only
+/* global $ */
+
+//= require govuk_publishing_components/dependencies
+
+//= require govuk_publishing_components/lib
+//= require govuk_publishing_components/lib/track-click
```

Comment:
Confusingly, `lib` is both a directory and a file, `lib.js`. I'd thought that this would include the whole directory, but that would need `require_tree`, so I think this is including `lib.js`. However, `lib.js` then includes everything in the `lib` directory, plus the polyfills.

Since `lib` the directory contains `track-click`, line 6 isn't needed?